ddtrace/tracer/option.go

```diff
@@ -1141,6 +1141,12 @@ func SpanType(name string) StartSpanOption {
 	return Tag(ext.SpanType, name)
 }
 
+func WithSpanLinks(links []ddtrace.SpanLink) StartSpanOption {
+	return func(cfg *ddtrace.StartSpanConfig) {
+		cfg.Links = append(cfg.Links, links...)
+	}
+}
+
```

Comment:
This function is missing unit tests.

Reply:
I pushed the tests right after you review 😓